packages/theming/src/elements/ThemeProvider.tsx

```diff
@@ -13,7 +13,19 @@ import DEFAULT_THEME from './theme';
 import { useDocument } from '../utils/useDocument';
 
 interface IGardenThemeProviderProps extends Partial<ThemeProps<DefaultTheme>> {
-  focusVisibleRef?: React.RefObject<HTMLElement>;
+  /**
+   * Provides values for component styling. See styled-components
+   * [`ThemeProvider`](https://styled-components.com/docs/api#themeprovider)
+   * for details.
+   */
+  theme?: DefaultTheme;
```

Comment:
This repeats the definition from the extended interface. But without it, we aren't able to document the prop. Therefore, I think it's imperative in this one case to make an exception and include it.